lib/ansible/cli/galaxy.py

```diff
@@ -469,11 +469,34 @@ def add_install_options(self, parser, parents=None):
                                  'collection. This will not ignore dependency conflict errors.'
         else:
             args_kwargs['help'] = 'Role name, URL or tar file'
+            if self._implicit_role:
+                args_kwargs['help'] += '. This is mutually exclusive with --requirements-file.'
+            else:
+                args_kwargs['help'] += '. This is mutually exclusive with --role-file.'
             ignore_errors_help = 'Ignore errors and continue with the next specified role.'
 
+        if self._implicit_role:
+            # might install both roles and collections
+            description_text = (
+                'Install roles and collections from file(s), URL(s) or Ansible '
+                'Galaxy to the first entry in the config COLLECTIONS_PATH for collections '
+                'and first entry in the config ROLES_PATH for roles. '
+                'The first entry in the config ROLES_PATH can be overridden by --roles-path '
+                'or -p, but this will result in only roles being installed.'
+            )
+            prog = 'ansible-galaxy install '
```

Comment:
The spaces in the usage output seems to be handled by `argparse` automatically so this trailing space results in two space being added (between `install` and `[-h]`):
`usage: ansible-galaxy install  [-h]...`

Reply:
Good spotting, I totally missed it.